src/commands/functions/create.js

```diff
@@ -197,22 +197,32 @@ const pickTemplate = async function () {
 const DEFAULT_PRIORITY = 999
 
 /* get functions dir (and make it if necessary) */
-const ensureFunctionDirExists = function (context, flags, config) {
-  const functionsDir = config.build && config.build.functions
-  if (!functionsDir) {
-    context.log(`${NETLIFYDEVLOG} No functions folder specified in netlify.toml`)
-    process.exit(1)
+const ensureFunctionDirExists = async function (context, flags, config) {
+  let functionsDirHolder = config.build && config.build.functions
+  if (!functionsDirHolder) {
+    context.log(`${NETLIFYDEVLOG} functions folder not specified in netlify.toml or UI settings`)
+    const { functionsDir } = await inquirer.prompt([
+      {
+        type: 'input',
+        name: 'functionsDir',
+        message: 'Enter the path where your functions should live: netlify/functions:',
+        default: 'netlify/functions',
+      },
+    ])
+
+    functionsDirHolder = functionsDir
```

Comment:
Should we update the user's site settings with the value retrieved from the user?
If the user choses `my_netlify_functions` as the functions directory how will our build system know about it?

Reply:
`netlify/functions` and `functions` directories should be detected by bitballoon and added as the site's `functions_dir`, but to ensure it's in the CLI too, I'm going to update the `netlify.toml` to have the new functions directory.

Reply:
> I'm going to update the `netlify.toml` to have the new functions directory.

Please consider that some sites don't have a `netlify.toml`. In that case it might be better to use an `api` call to update the site settings.

If the functions directory is the default one `netlify/functions` that won't be necessary as you said.

Reply:
Ahh, got it! I'll do that then, thanks 😊

Reply:
Seems like it might make sense after this to add a `functions:set` command that updates the functions directory

Reply:
nvm we already have that :P